### DISCUSSION TITLE
Add label components

### DISCUSSION THREAD

**Discussion by HichOoMa** [saadeghi/daisyui#2481] [Ideas / feature requests]
I want to create label components with several designs and actions

**Comment by saadeghi** [comment]
Please explain more. And attach some photos if possible.

**Comment by HichOoMa** [comment]
This an example of what I think of in a [Figma design](https://www.figma.com/file/hfdL8367BIaDKvR7o1zfRD/Untitled?type=design&node-id=0%3A1&mode=design&t=Hh18YBGH2VsIgXvt-1)
NB: I am not familiar with figma so I cannot stimulate the hover action :laughing:
